emloop/cli/ls.py

```diff
@@ -86,15 +85,15 @@ def get_model_name(config: dict) -> str:
     return config['model']['name'] if 'model' in config and 'name' in config['model'] else 'Unnamed'
 
 
-def _print_trainings_long(trainings: Iterable[Tuple[str, dict, TrainingTrace]]) -> None:
+def _print_trainings_long(trainings: Iterable[Tuple[str, dict, dict]]) -> None:
```

Comment:
I have changed this, such that the config is loaded as dict, but the keys used are not strings anymore but for example `TrainingTraceKeys.EPOCHS_DONE`. If you are not ok with this, I will rewrite it using custom class instead of dict.